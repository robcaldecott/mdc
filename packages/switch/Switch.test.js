import React from "react";
import { Switch } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(renderer.create(<Switch defaultChecked />)).toMatchSnapshot();
});
