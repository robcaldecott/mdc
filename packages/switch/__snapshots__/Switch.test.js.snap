// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="mdc-switch"
>
  <input
    className="mdc-switch__native-control"
    defaultChecked={true}
    type="checkbox"
  />
  <div
    className="mdc-switch__background"
  >
    <div
      className="mdc-switch__knob"
    />
  </div>
</div>
`;
